src/components/InsufficientDataDialog.tsx

```diff
@@ -0,0 +1,53 @@
+import {
+  Button,
+  Heading,
+  Modal,
+  ModalBody,
+  ModalCloseButton,
+  ModalContent,
+  ModalFooter,
+  ModalOverlay,
+  Text,
+  VStack,
+} from "@chakra-ui/react";
+import { FormattedMessage } from "react-intl";
+
+interface InsufficientDataDialogProps {
+  onClose: () => void;
+}
+
+const InsufficientDataDialog = ({ onClose }: InsufficientDataDialogProps) => {
```

Comment:
When does this happen now? The "Train model" button is disabled when you have insufficient samples.

Reply:
I'm not sure what the right behaviour is here - I think it overlaps with the onboarding/hints discussion.

Reply:
Good point [7dfffdf](https://github.com/microbit-foundation/ml-trainer/pull/299/commits/7dfffdfa7c77481ee4a2ba63568e568831084745)